spec/models/odk/response_pattern_parser_spec.rb

```diff
@@ -6,28 +6,31 @@
   subject(:output) { described_class.new(pattern, src_item: src_item).to_odk }
 
   describe "xpath route handling" do
-    let(:q1) { Odk::QingDecorator.decorate(form.sorted_children[0]) }
-    let(:g2) { Odk::QingGroupDecorator.decorate(form.sorted_children[1]) }
-    let(:q21) { Odk::QingDecorator.decorate(form.sorted_children[1].sorted_children[0]) }
-    let(:q22) { Odk::QingDecorator.decorate(form.sorted_children[1].sorted_children[1]) }
-    let(:g3) { Odk::QingGroupDecorator.decorate(form.sorted_children[2]) }
-    let(:q31) { Odk::QingDecorator.decorate(form.sorted_children[2].sorted_children[0]) }
-    let(:q31a) { Odk::QingDecorator.decorate(form.sorted_children[2].sorted_children[0]).subqings.first }
+    let(:q1) { Odk::QingDecorator.decorate(form.c[0]) }
+    let(:g2) { Odk::QingGroupDecorator.decorate(form.c[1]) }
+    let(:q21) { Odk::QingDecorator.decorate(form.c[1].c[0]) }
```

Comment:
Does it make sense to have some sort of helper for selecting these various children?